packages/examples/cvat/exchange-oracle/src/services/cloud/s3.py

```diff
@@ -8,6 +9,8 @@
 from src.services.cloud.client import StorageClient
 
 DEFAULT_S3_HOST = "s3.amazonaws.com"
+if TYPE_CHECKING:
+    from mypy_boto3_s3.client import S3Client
```

Comment:
```suggestion
    from mypy_boto3_s3.client import _S3Client
```
Probably, it's not a good idea to have 2 different symbols with the same name in the same file.

Reply:
Ah, good catch!